### ISSUE TITLE
Add ability to paste in a list of urls

### ISSUE THREAD

**Issue by raelb** [Nenirey/AWGG#11]
Hi,
It would be useful if I could paste in a list of urls to be downloaded. For example having a dialog like this:

![image](https://user-images.githubusercontent.com/13108198/99155793-e953e600-26c3-11eb-8f92-0c8ecdaa8680.png)

Thanks.

**Comment by Nenirey** [comment]
Yes, am thinking in some like import from clipboard function

**Comment by raelb** [comment]
Yeah that would be useful.  The advantage of a separate dialog is that you could also have an option to specify download folder. I sometimes have a need to download a batch of files into a specific folder, so that would be useful. Thanks
